packages/sage/src/commands/export.ts

```diff
@@ -227,3 +243,11 @@ async function exportModule(module: string, project: ProjectInfo, commandOptions
 		logger.ready(`Successfully added plugin "${color.bold(packageName)}" to your Robo!`)
 	}
 }
+
+async function isUsingWorkspacesAndNpm(packageManager: string, hasWorkspaces: boolean): Promise<string> {
```

Comment:
Looks good but this function could use better naming for future-proofing. There may be other package managers (Bun?) that may also have similar issues requiring the use of `install-local`. May I suggest `usesLocalWorkaround()` or something like it? I dunno if "uses" is the right word for this, but something implying "local workaround" may work better for stuff unrelated to NPM and workspaces.